### ISSUE TITLE
MKL components are not installed when make sysinstall

### ISSUE THREAD

**Issue by Quallyjiang** [NervanaSystems/neon#386]
MKL install script is not included in sysinstall sections. Is that desired? Means MKL is not allowed to be installed in system wide?

env: $(MKL_ENGINE) $(DATA_LOADER) $(ACTIVATE)
...
$(MKL_ENGINE):
        @echo "Building MKL Engine..."
        @./install_mkl.sh intel
...
sysinstall_nodeps: $(DATA_LOADER) neon_install
sysinstall: sysdeps $(DATA_LOADER) neon_install

**Comment by wei-v-wang** [comment]
@Quallyjiang  Thank you for trying neon! 
You are right about MKL components not installed when "make sysinstall". It it a bug that we will make sure we address it in our next release. 

You may already know, you can make the following changes to install MKL components as well during sysinstall: 

-sysinstall: sysdeps $(DATA_LOADER) neon_install
+sysinstall: $(MKL_ENGINE) sysdeps $(DATA_LOADER) neon_install

So in conclusion, MKL component is definitely allowed to be installed system-wide

**Comment by wei-v-wang** [comment]
If neon is to be shared by multiple users, the suggestion is to put neon directory on a shared location, the purpose is for all users to be able to locate neon's MKL library.  Please let us know if you encounter further issues.

**Comment by Quallyjiang** [comment]
thanks!

**Comment by pabutcher** [comment]
If we have already completed a system-wide install, how do we install the MKL library?

**Comment by wei-v-wang** [comment]
The way I did was:
1) "make sysuninstall"
2) Add the $(MKL_ENGINE) target to sysinstall 
3) "make sysinstall" 

Alternatively, I think you can also try the following if you do not want to do "make sysuninstall": 
./install_mkl.sh
